nab_evaluation.py

```diff
@@ -0,0 +1,144 @@
+"""
+    Evaluation module for point-wise prediction algorithms in terms of NAB (Numenta Anomaly Benchmark) score.
+"""
+import numpy as np
+
+def scaledSigmoid(relativePositionInWindow):
+    if relativePositionInWindow > 3.0:
```

Comment:
Will it ever happen? The condition for this function to be called is that the prediction point in properly within the window, right?

Reply:
Yes, if the point is predicted right outside the window, the scaled sigmoid is used to calculate its score.